src/nodemanagerbase.cpp

```diff
@@ -184,25 +184,21 @@ UA_LocalizedText make_localised( UaString text )
     out.text = *(new UaString(text))->impl();
     return out;
 }
-    
+
 UaStatus NodeManagerBase::addNodeAndReference( 
     UaNode* parent,
     UaNode* to,
     const UaNodeId& refType)
 {
-   // std::cout << __FUNCTION__ << "from.id=" << from.toString().toUtf8() << " to.id=" << to->nodeId().toString().toUtf8() <<  std::endl;
```

Comment:
Add a LogIt statement here - I assume if this line was important enough to merit a std::cout it should probably also be loggable?

Reply:
Hi Ben,
I removed this line completely in this commit, I think there is nothing to fix here. I don't think it deserves a log from here.